dotnet/samples/KernelSyntaxExamples/Example80_OpenAIFiles.cs

```diff
@@ -49,7 +50,7 @@ public async Task RunFileLifecycleAsync()
         WriteLine("SOURCE:");
         WriteLine($"# Name: {fileReference.FileName}");
         WriteLine("# Content:");
-        WriteLine(await fileContent.GetContentAsync());
+        WriteLine(Encoding.UTF8.GetString((await fileContent.GetContentAsync()).ToArray()));
```

Comment:
This and others like it should be able to use .Span rather than .ToArray.

Reply:
Thanks, fixed.